install/pi/install.sh

```diff
@@ -40,8 +40,8 @@ source ~/env/bin/activate
 #make sure the virtual environment is active
 source ~/env/bin/activate
 
-# install pandas and numpy
+# install pandas and numpy and Adafruit CircuitPython
 pip install pandas #also installs numpy
-
+pip install adafruit-circuitpython-lis3dh
```

Comment:
Can we not install this with `pip` by adding it into the `.[pi]` section of `setup.py`? I don't think this install file is used.